zenodo/modules/spam/templates/zenodo_spam/safelist/admin.html

```diff
@@ -49,17 +67,30 @@ <h1>Safelist admin</h1>
             <th>Name</th>
             <th>External</th>
             <th>Records</th>
+            <th>GT</th>
+            <th>Descriptions</th>
           </tr>
         </thead>
         <tbody>
           {% for user in users.values() %}
             <tr id="{{ user.id }}">
-              <td><a href="{{ url_for('zenodo_spam.delete', user_id=user.id) }}">{{ user.id }}</a></td>
-              <td>{{ user.username }}</td>
-              <td>{{ user.email }}</td>
-              <td>{{ user.full_name }}</td>
-              <td>{{ user.external | join(', ') }}</td>
-              <td>{{ user.last_records }}</td>
+              <td>
+                <a target="_blank" href="{{ url_for
+                ('zenodo_spam.delete', user_id=user.id) }}">{{ user.id }}</a>
+              </td>
+              <td class="selectable">{{ user.username }}</td>
+              <td class="selectable">{{ user.email }}</td>
+              <td class="selectable">{{ user.full_name }}</td>
+              <td class="selectable">{{ user.external | join(', ') }}</td>
+              <td class="selectable">{{ user.last_records }}</td>
```

Comment:
Comment: Can you make a link to the record itself? That would save me quite some time in determining if a user is spammer.